## Conclusion

In [2]:
!pip install prettytable

Defaulting to user installation because normal site-packages is not writeable
  Obtaining dependency information for prettytable from https://files.pythonhosted.org/packages/c5/16/ec5cc65437dce97d2814a7ba31842b0ee958d102f6e99e264c35f15c328f/prettytable-3.10.2-py3-none-any.whl.metadata


DEPRECATION: Loading egg at c:\programdata\anaconda3\lib\site-packages\vboxapi-1.0-py3.11.egg is deprecated. pip 23.3 will enforce this behaviour change. A possible replacement is to use pip for package installation..


In [3]:
from prettytable import PrettyTable

In [5]:
ptable1 = PrettyTable()
ptable1.title = " Model Comparision "
ptable1.field_names = ['Model Name','Hyperparameter Tunning', 'Accuracy']

# *****************************************************************

ptable2 = PrettyTable()
ptable2.title = " Model Comparision "
ptable2.field_names = ['Model Name','Hyperparameter Tunning', 'categorical_crossentropy', 'Accuracy']

print("\n\n ********************************* Deep Learning LSTM Model Comparision ***********************************")
ptable2.add_row(["LSTM With 1_Layer(neurons:32)","Done","0.47", "0.90"])
ptable2.add_row(["LSTM With 2_Layer(neurons:48, neurons:32)","Done","0.39", "0.90"])
ptable2.add_row(["LSTM With 2_Layer(neurons:64, neurons:48)","Done","0.27", "0.91"])

print(ptable2)



 ********************************* Deep Learning LSTM Model Comparision ***********************************
+----------------------------------------------------------------------------------------------------------+
|                                            Model Comparision                                             |
+-------------------------------------------+------------------------+--------------------------+----------+
|                 Model Name                | Hyperparameter Tunning | categorical_crossentropy | Accuracy |
+-------------------------------------------+------------------------+--------------------------+----------+
|       LSTM With 1_Layer(neurons:32)       |          Done          |           0.47           |   0.90   |
| LSTM With 2_Layer(neurons:48, neurons:32) |          Done          |           0.39           |   0.90   |
| LSTM With 2_Layer(neurons:64, neurons:48) |          Done          |           0.27           |   0.91   |
+-----------------

#### Deep Learning LSTM Model Comparision : When we talking about LSTM Model, here with LSTM we are using simple RAW data, LSTM perform very-very well and got highest 91% accuracy with 2_layer LSTM with hyperparameter Tunning and also we can clearly when we are increasing LSTM layer and Hyperparameter Tunning the cross-entropy value is decreasing and Accuracy is increasing.